ui-templates/custom-types/unit_data.json

```diff
@@ -0,0 +1,21 @@
+{
+    "unit_data": {
+        "python::module": "simulariumio",
+        "python::object": "UnitData",
+        "parameters": {
+            "name": {
+                "name": "Name",
+                "data_type": "text",
+                "description": "Provide the unit name (we support this list https://github.com/hgrecco/pint/blob/master/pint/default_en.txt)",
```

Comment:
shouldn't this be data_type `enum` instead of `text`?

Reply:
Like you did here: 
```     
 "options": [
                    "SPHERE",
                    "PDB",
                    "OBJ",
                    "FIBER"
                ]
```

Reply:
@blairlyons  change this file to two files, one for time and one for space, which each have options of the available SI units for time and space